content/_help/verify-your-identity/verify-your-address-by-mail._en.md

```diff
@@ -10,7 +10,7 @@ redirect_from:
   - /help/creating-an-account/how-do-i-verify-my-identity-by-mail/
 ---
 
-Identity verification is the process where you prove you are you - and not someone pretending to be you.  If you cannot [verify your phone number](/help/verify-your-identity/phone-number/), you can use your address to verify by mail instead. You will get a letter in the mail with a verification code to finish verifying your identity.
+In some cases, you can verify your address by mail instead of [verifying your phone number.](/help/verify-your-identity/phone-number/) You will get a letter in the mail with a verification code to finish verifying your identity.
```

Comment:
Confession: I'm personally not a big fan of striking this first sentence, but it's what the professionals dealing with content requested so I totally defer to their expertise and did so.